ui.frontend/html/src/components/AddContactView/index.jsx

```diff
@@ -25,47 +26,41 @@ class AddContactView extends React.Component {
     this.fields[field] = value
   }
 
-  componentDidMount () {
+  componentDidMount() {
     // Expose this view globally to be accessible for Java
     window.view = this
   }
 
-  render () {
+  render() {
     return (
-      <div className='form-horizontal' id='add-contact-form'>
-        <div className='form-group'>
-          <label className='col-sm-2 control-label'>
-            <Text message='label.jid' />
-          </label>
-          <div className='col-sm-10'>
-            <input
-              autoFocus
-              type='text'
-              value={this.fields.jid}
-              onChange={this.onChangeField}
-              name='jid'
-            />
-          </div>
-        </div>
-        <div className='form-group'>
-          <label className='col-sm-2 control-label'>
-            <Text message='label.nickname' />
-          </label>
-          <div className='col-sm-10'>
-            <input type='text' value={this.fields.displayName} onChange={this.onChangeField} name='displayName' />
-          </div>
-        </div>
-        <div className='form-group'>
-          <div className='col-sm-offset-2 col-sm-10'>
-            <button id='cancel-add-contact' onClick={this.props.mainUI.doCancelAddContact} className='cancel-add-contact-btn btn btn-default'>
-              <Text message='action.cancel' />
-            </button>
-            <button id='add-contact' onClick={this.onClickSubmit} className='submit-add-contact btn btn-primary'>
-              <Text message='action.addContact' />
-            </button>
-          </div>
-        </div>
-      </div>
+      <Row id='add-contact-form'>
+        <Col sm={12} md={{offset: 1, span: 10}} lg={{ offset: 2, span: 8 }}>
+          <Form>
+            <Form.Group>
+              <Form.Label><Text message='label.jid' /></Form.Label>
+              <Form.Control autoFocus type='text' placeholder='Contact JID'
```

Comment:
The problem is that our React-Localize returns a react component, which cannot be placed in the placeholder since it only accepts a string.

as a work around in the latest patch I imported the dictionary directly and accessed the needed value.